src/encoded/static/components/cart/cart.js

```diff
@@ -82,9 +97,22 @@ const requestedFacetFields = displayedFacetFields.concat([
     { field: '@id' },
     { field: 'file_format_type' },
     { field: 'title' },
+    { field: 'href' },
+    { field: 'dataset' },
+    { field: 'biological_replicates' },
 ]);
 
 
+/**
+ * Maps long annotation_type values to shorter versions for Valis track labels. Any not included
+ * remain unchanged.
+ */
+const annotationTypeMap = {
+    'candidate Cis-Regulatory Elements': 'cCRE',
+    'representative DNase hypersensitivity sites (rDHSs)': 'rDHS',
+};
```

Comment:
Maybe we will want to change the labels on the annotation pages to match those in the cart and then this could go in globals.js or objectutils.js.

Reply:
I moved this object to genome_browser.js, since it seems unlikely this module would import from another module that imports from it (not that this would cause problems, as we found several months ago; that still makes me uncomfortable).